instagram-app/src/components/CommentSection/CommentSection.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import Moment from 'react-moment'
+
+import './CommentSectionStyles.css';
+
+class CommentSection extends React.Component {
+    constructor() {
```

Comment:
since there is no state here you could leave out the constructor and super entirely.
this component could alternatively be written as a stateless function component.